includes/class-wsuwp-scholarships.php

```diff
@@ -51,6 +51,13 @@ class WSUWP_Scholarships {
 	public $taxonomy_slug_ethnicity = 'ethnicity';
 
 	/**
+	 * @since 0.0.3
+	 *
+	 * @var string Slug for tracking the Grade Level taxonomy.
+	 */
+	public $taxonomy_slug_grade = 'grade';
```

Comment:
I'm a little worried about the non-prefixed `grade` only because it could cause an unknown issue if for some reason we added a system wide `grade` taxonomy. I doubt that we would actually do that, but it may be good to be defensive here and stick with `scholarship_grade` or `sch_grade` or `sfs_grade` or something along those lines.